### ISSUE TITLE
pnext: SqlProcessor unittests fails.

### ISSUE THREAD

**Issue by chrhol** [Starcounter/Starcounter#3241]
All unittests currently in project `Starcounter.SqlProcessor.Tests` fails with the same error:

> An unhandled System.Exception was thrown while executing this test : ScErrThreadNotAttached (SCERR1004): The operation failed because the current thread wasn't attached to the scheduler.
Version: 2.0.0.
Help page: https://github.com/Starcounter/Starcounter/wiki/SCERR1004.

Should these tests still run as unittests? It looks like some changes in pnext requires them to be run inside a scheduler thread now.

I will disable them for the time being (`MultithreadedTest`, `SequentialTest`, `PerformanceTests`, `HelloProcessor` and `SqlSyntax`) and enable unittests to run on daily and nightly builds for pnext.

**Comment by k-rus** [comment]
Thank you for reporting!
The tests are supposed to work. I guess they were not adopted to the threading model of pnext.

**Comment by k-rus** [comment]
I checked that the test stopped working due to API change. Part of the test already runs as a native unit test of SQL parser. Another part will be added there. Then this test will be completely removed. Now I will completely disable it.

**Comment by k-rus** [comment]
> Now I will completely disable it.

If I see correctly, it is already not in use.

**Comment by chrhol** [comment]
> If I see correctly, it is already not in use.

Yes, I removed the `Test` attribute so they are not running as unittests. However if they are running as native unittests now maybe we should consider deleting the code altogether just to clean up unusued code a bit.

**Comment by k-rus** [comment]
> If I see correctly, it is already not in use.
> Yes, I removed the Test attribute so they are not running as unittests. However if they are running as native unittests now maybe we should consider deleting the code altogether just to clean up unusued code a bit.

I will remove entire the test project as soon as test is completely replicated in native code.
